go/enclave/rpcencryptionmanager/rpc_encryption_manager.go

```diff
@@ -137,15 +135,20 @@ func (rpc *RPCEncryptionManager) EncryptTxReceiptWithViewingKey(address gethcomm
 
 // DecryptTx decrypts an L2 transaction encrypted with the enclave's public key.
 func (rpc *RPCEncryptionManager) DecryptTx(encryptedTx common.EncryptedTx) (*common.L2Tx, error) {
```

Comment:
The logic is now more complex, but that's just how `eth_sendRawTransaction` requests are formatted.